deps.bzl

```diff
@@ -3004,7 +2998,6 @@ def prysm_deps():
         name = "com_github_wealdtech_go_eth2_types_v2",
         build_directives = [
             "gazelle:resolve go github.com/herumi/bls-eth-go-binary/bls @herumi_bls_eth_go_binary//:go_default_library",
-            "gazelle:resolve go github.com/prysmaticlabs/go-ssz @com_github_prysmaticlabs_go_ssz//:go_default_library",
```

Comment:
I think you need to leave this. It's part of `go-eth2-types.v2`

Reply:
I took a look through and it seems like its not needed anymore.